### DISCUSSION TITLE
FungalTraits database Species Hypothesis (SH) level functional assignment

### DISCUSSION THREAD

**Discussion by YiweiDuan01** [ChiLiubio/microeco#365] [Q&A] [answered]
Hi Chi,
Thanks for creating such a neat package. I am trying to run the trans_func$cal_spe_func command on fungal ITS sequencing results, and was having a hard time figuring out whether and how I can use the SH level functional assignment instead of the genus level functional assignment (which I believe it is now the default?). 

According the creators of FungalTraits, both genus level and SH level functional assignment is possible (https://doi.org/10.1007/s13225-020-00466-2), is this something you've also incorporated into the microeco package? 

If you could elaborate on this, I would greatly appreciate it! 

Best regards,
Yiwei

**Comment by ChiLiubio** [comment, marked as answer]
Hi Yiwei,
Thanks. Yes. The implemented method is genus level assignment. The main reason is there is generally few exact species information in taxonomic assignment result of fungal sequencing data. If we discard the ASV/OTU that has no species names, the result will be hard to explain. Another reason is the trait of fungal genera used in these databases is generally conservative, which is more suitable for the trait prediction on genus level, which can make the result robust in my view.

Best,
Chi

**Reply by YiweiDuan01** [reply]
Hi Chi,
Thanks for your reply. Clear! 

Best,
Yiwei
